zds/tutorialv2/views/display.py

```diff
@@ -84,6 +85,9 @@ def get_context_data(self, **kwargs):
             queryset_pagination = PublishedContent.objects.filter(
                 content_type=self.current_content_type, must_redirect=False
             )
+            
+            if self.current_content_type == 'OPINION':
```

Comment:
Black (un *linter* pour uniformiser le code) râle encore, j'imagine que c'est à cause de ces guillemets là
```suggestion
            if self.current_content_type == "OPINION":
```